airmozilla/main/static/main/js/calendar.js

```diff
@@ -37,7 +42,10 @@ $(function() {
     startsOnMonday.on('change', function(){
         localStorage.setItem('startsOnMonday', startsOnMonday.prop('checked')? 1 : 0);
         container.fullCalendar('destroy');
+        view = localStorage.getItem('view');
         initCalendar();
+        $('.fc-right button').on('click', save_view);
     });
     initCalendar();
+    $('.fc-right button').on('click', save_view);
```

Comment:
http://fullcalendar.io/docs/display/viewRender/
"Triggered when a new date-range is rendered, or when the view type switches."